src/test/java/core/basesyntax/service/ReportServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportServiceTest {
+    private static ReportService reportService;
+
+    @BeforeAll
+    static void beforeAll() {
+        reportService = new ReportService();
```

Comment:
I think that `ReportService ` should be an interface and it should had it's implementation in order to follow style of rest of your code.
```suggestion
        reportService = new ReportServiceImpl();
```